importexport/musicxml/importmxmlpass2.cpp

```diff
@@ -5632,12 +5683,12 @@ void MusicXMLParserNotations::mordentNormalOrInverted()
       {
       Q_ASSERT(_e.isStartElement() && (_e.name() == "mordent" || _e.name() == "inverted-mordent"));
 
-      Notation notation { _e.name().toString() };
+      GenericAnnotation notation { _e.name().toString(), "ornaments" };
```

Comment:
There is a lot of duplicate code in creating the notation / generic annotation and adding the parameters to it. Please create a convernience function. Note I am aware I created this issue myself, but you increased the number of occurrences.